packages/wallets/src/components/Dropzone/Dropzone.tsx

```diff
@@ -137,21 +137,36 @@ const Dropzone: React.FC<TProps> = ({
                     )}
                     {!showHoverMessage && file && (
                         <React.Fragment key={file.name}>
-                            <div
-                                className={classNames('wallets-dropzone__thumb', {
-                                    'wallets-dropzone__thumb--has-frame': hasFrame,
-                                })}
-                                style={{ backgroundImage: `url(${file.preview})` }}
-                            >
-                                {hasFrame && <DropzoneFrame />}
-                                <IconButton
-                                    className='wallets-dropzone__remove-file'
-                                    data-testid='dt_remove-button'
-                                    icon={<CloseIcon width={12} />}
-                                    onClick={removeFile}
-                                    size='sm'
-                                />
-                            </div>
+                            {file.file.type.indexOf('pdf') !== -1 ? (
+                                <div className='wallets-dropzone__thumb wallets-dropzone__thumb--pdf'>
+                                    <IconButton
+                                        className='wallets-dropzone__remove-file'
+                                        data-testid='dt_remove-button'
+                                        icon={<CloseIcon width={12} />}
```

Comment:
can we use icons from `deriv/quill-icons` instead

Reply:
@aizad-deriv For these we need to bump up the quill-icons version to use the LegacyClose icon as the current available icons causes [issues](https://deriv-group.slack.com/archives/C06BKDYUBLK/p1713843525635559) when being used. We can do this in a separate PR

Reply:
Alright let's just add a TODO comment so we can refer back to this once it is added 👍🏼

Reply:
added TODO 🙏